### ISSUE TITLE
跑出来的结果比较差，能帮忙看一下吗？

### ISSUE THREAD

**Issue by PGogo** [zju3dv/Mirrored-Human#12]
谢谢你们的开源，这个想法挺巧妙的！但是我在尝试跑一下代码后发现挺多问题的。
除了在这个issue #9 里面提到的问题，还有https://github.com/zju3dv/EasyMocap/blob/2e83642c7c95f777eeec5cb1f49386c5f8e792f4/easymocap/estimator/SPIN/spin_api.py#L291 这里面少了一个body_params['Th']，我尝试将它补充为 body_params['Th'] = np.zeros_like(pred_camera.detach().cpu().numpy()) 后能跑起来，但是得到的结果是这样的
![000000](https://user-images.githubusercontent.com/41355904/212876538-28940691-e8db-42b1-a500-aec7eb50e2df.jpg)
预训练模型用的是 #9 里面的模型。请问这样的情况怎么解决呢？另外训练代码还会公开吗？

**Comment by raypine** [comment]
你好，代码被重构了，文字说明忘改了，新的启动方式见这里的[easymocap文档](https://chingswy.github.io/easymocap-public-doc/quickstart/quickstart.html)
